guide/english/cplusplus/index.md

```diff
@@ -1,102 +1,81 @@
 ---
 title: C++
 ---
-# Hello World! - Your First C++ Program
+# C++
 
 ## What is C++?
 
-* C++ is a general purpose programming [Object-Oriented](https://en.wikipedia.org/wiki/Object-oriented_programming) language which has been used since the 1990's
-* It was designed as a thesis paper by [Bjarne Stroustrup](https://en.wikipedia.org/wiki/Bjarne_Stroustrup) under the name "C with Classes", which later became C++.
-* It is a version of C that includes Object-Oriented elements, including classes and functions.
-
-* It is considered one of the biggest programming languages, as you can see in the following image:
-![Img](http://static1.businessinsider.com/image/59deb30392406c21008b6148-1200/for-bonus-points-heres-the-chart-showing-these-languages-relative-popularity.jpg)
+- C++ is a general purpose programming [object-oriented programming](https://en.wikipedia.org/wiki/Object-oriented_programming) (OOP) language which has been used since the 1990's
+- It was designed as a thesis paper by [Bjarne Stroustrup](https://en.wikipedia.org/wiki/Bjarne_Stroustrup) under the name "C with classes", which later became C++.
+- It is a version of C that includes OOP elements, including classes and functions.
+- It is considered one of the biggest programming languages, as you can see in the following image:
+![programming language popularity](http://static1.businessinsider.com/image/59deb30392406c21008b6148-1200/for-bonus-points-heres-the-chart-showing-these-languages-relative-popularity.jpg)
```

Comment:
@cmccormack I think we can put this in unnecessary parts too, we do not absolutely need this image to be here and this article could do just fine without it

Reply:
Are you referring to the last bullet only or the entire section?

Reply:
Just the graphic

Reply:
Removed, thanks